test/encase2_.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var uncurry2 = require('ramda').uncurryN(2);
+
+var eq = require('./utils').eq;
+var rem = uncurry2(require('./utils').rem);
+var S = require('..');
```

Comment:
I'd prefer not to intermingle imports and other code. I suggest:

```javascript
'use strict';

var R = require('ramda');

var S = require('..');
var utils = require('./utils');


var eq = utils.eq;
var rem = R.uncurryN(2, utils.rem);
```

I realize you're simply doing as we do in other files, though, so you needn't change this.